packages/client-sdk-nodejs/package.json

```diff
@@ -15,12 +15,15 @@
   "scripts": {
     "prebuild": "eslint . --ext .ts",
     "test": "jest --testPathIgnorePatterns auth-client.test.ts --maxWorkers 1",
-    "integration-test-auth": "jest auth-client.test.ts --maxWorkers 1",
-    "integration-test-store": "jest storage.test.ts --maxWorkers 1",
-    "unit-test": "jest unit",
-    "integration-test-leaderboard": "jest leaderboard --maxWorkers 1",
-    "integration-test-control-cache-topics": "jest integration --testPathIgnorePatterns \"auth-client.test.ts|leaderboard.test.ts|storage.test.ts\" --maxWorkers 1",
+    "integration-test-auth": "jest auth/ --maxWorkers 1",
+    "integration-test-cache": "jest cache/ --maxWorkers 1",
+    "integration-test-control-cache-topics": "npm run integration-test-cache && npm run integration-test-topics",
+    "integration-test-leaderboard": "jest leaderboard/ --maxWorkers 1",
+    "integration-test-storage": "jest storage/ --maxWorkers 1",
+    "integration-test-store": "npm run integration-test-storage",
```

Comment:
The canaries reference this script, so we leave it in place for now (but updated) to not break it.